src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 200) {
```

Comment:
[CODE ISSUE] The default value of health should be 100, not 200. This does not align with the task requirements.